tailwind.config.js

```diff
@@ -20,6 +20,10 @@ module.exports = {
   darkMode: ["class"],
   theme: {
     extend: {
+      screens: {
```

Comment:
Using pixels instead of rem in the Tailwind config solved the problem of ui breaking

Reply:
https://github.com/ohcnetwork/care_fe/pull/9013#issuecomment-2466112422